akanda/rug/test/unit/api/test_configuration.py

```diff
@@ -66,6 +66,15 @@ def __init__(self, id_, **kwargs):
     dns_nameservers=['8.8.8.8'],
     host_routes={})
 
+fake_subnet_with_slaac = FakeModel(
```

Comment:
So this my fault.  We should be using the real model Objects instead of fake ones.  If we had used them here, then the missing attributes cited above would have been caught.